src/main/java/Duke.java

```diff
@@ -1,10 +1,248 @@
+import Duke.exceptions.exceedNum;
+import Duke.tasks.Deadline;
+import Duke.tasks.Event;
+import Duke.tasks.Task;
+import Duke.tasks.Todo;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    public static void goodbye() {
+        System.out.println("\nBye.Hope to see you again soon!");
+    }
+
+    public static void greeting() {
+    /*String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+    }
+
+    public static void checkSum(int sum) throws exceedNum {
+        if (sum==100){
+            throw new exceedNum();
+        }
+    }
+
+    public static void numList(int sum) {
+        if (sum == 1) System.out.println("\tNow you have " + sum + " task in the list.\n");
+        else System.out.println("\tNow you have " + sum + " tasks in the list.\n");
+    }
+
+    public static void processDone(String answer, Task[] tasks) {
+        int index;
+        try {
+            String[] sentence = answer.split(" ");
+            index = Integer.parseInt(sentence[1]);
+            tasks[index - 1].markAsDone();
+        }
+        catch(NullPointerException e){
+            System.out.println("\n\tThis index of task does not exist.\n");
+        }
+        catch(NumberFormatException e){
+            System.out.println("\n\tPlease enter a number.\n");
+        }
+        catch(ArrayIndexOutOfBoundsException e){
+            System.out.println("\n\tPlease enter the index of the task.\n");
+        }
+    }
+
+    public static int processEvent(String answer, Task[] tasks, int sum) {
+        try{
```

Comment:
There is a space between `try` and `{`.

Reply:
The same applies to your `catch (...)` and `{` for all your `try-catch` blocks. Please take note of the spaces in-between.

Reply:
Thank you! I've added them.